apps/website/src/components/ui/InstallButton.tsx

```diff
@@ -4,30 +4,110 @@ import { Copy, CopyCheck } from 'lucide-react';
 import { useEffect, useState } from 'react';
 import { useCopyToClipboard } from 'usehooks-ts';
 
+const getCommand = (manager: string) => {
+	switch (manager) {
+		case 'pnpm':
+			return 'pnpm install discord.js';
+		case 'yarn':
+			return 'yarn add discord.js';
+		case 'bun':
+			return 'bun add discord.js';
+		default:
+			return 'npm install discord.js';
+	}
+};
```

Comment:
You could just make an array of `{ packageManager: string; command: string }` or something like that, and then loop over that to generate the list items